### PR TITLE
flipped version

### PR DESCRIPTION
**Pull request by sjolicoeur**

### CODE REVIEW

File: amazon_payments/__init__.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
-from api import AmazonPaymentsAPI, AmazonPaymentsAPIError  # noqa
+from pkg_resources import get_distribution
+
+__version__ = get_distribution('django-oscar-amazon-payments').version
+
+VERSION = __version__
+
+try:
```
Comment (sjolicoeur) [inline amazon_payments/__init__.py]: @monokrome @pdufour the try might be overkill at this point as there won't be errors related to the import at setup

File: amazon_payments/__init__.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
-from api import AmazonPaymentsAPI, AmazonPaymentsAPIError  # noqa
+from pkg_resources import get_distribution
+
+__version__ = get_distribution('django-oscar-amazon-payments').version
+
+VERSION = __version__
```
Comment (sjolicoeur) [inline amazon_payments/__init__.py:5]: @monokrome @pdufour 

results into:

```
>>> from amazon_payments import VERSION
>>> VERSION
'0.2.1'
>>> from amazon_payments import __version__
>>> print __version__
0.2.1
```

File: amazon_payments/__init__.py (hunk 1 above)
Comment (sjolicoeur) [inline amazon_payments/__init__.py]: @pdufour  so you vote to remove it?

**Comment by pdufour** [comment]
@sjolicoeur do you know how to run the tests? want to make sure they still work.

**Comment by pdufour** [comment]
All working for me. :+1:

**Pull request merged**